tests/e2e/specs/volunteer-dashboard-tests.js

```diff
@@ -0,0 +1,177 @@
+describe("Volunteer dashboard", () => {
+  before(function() {
+    cy.fixture("users/volunteer3").as("openVolunteer");
+  });
+
+  describe("Open Volunteer", () => {
+    describe("Photo ID upload", () => {
+      beforeEach(function() {
+        cy.login(this.openVolunteer);
+        cy.server();
+        cy.visit("/");
+
+        cy.contains("Proof of identity").click();
+
+        cy.get("input[type=file]").attachFile("images/placeholder.jpg");
+      });
+
+      it("Should upload a photo id", function() {
+        cy.route("/api/user/volunteer-approval/photo-url", {
+          success: true,
+          message: "AWS SDK S3 pre-signed URL generated successfully",
+          uploadUrl: "noop"
```

Comment:
Is this just a random string that fails on client PUT?

Reply:
Yup, it just resolves to localhost:8080/noop